### PR TITLE
fomo framework (new)

### PR DESCRIPTION
**Pull request by amirfaramarzi**

### CODE REVIEW

**Comment by amirfaramarzi** [comment]
hi @waghanza 

what is the problem?

File: php/fomo/config/server.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+return [
+    'mode' => SWOOLE_BASE ,
+    'host' => '127.0.0.1',
```
Comment (waghanza) [inline php/fomo/config/server.php:5]:
```suggestion
    'host' => '0.0.0.0',
```

File: php/fomo/config/server.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    'port' => 9000 ,
```
Comment (waghanza) [inline php/fomo/config/server.php:6]:
```suggestion
    'port' => 3000 ,
```

File: php/fomo/config/server.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    'sockType' => SWOOLE_SOCK_TCP ,
+    'additional' => [
+        'worker_num' => cpuCount() * 2 ,
```
Comment (waghanza) [inline php/fomo/config/server.php:9]: maybe

```suggestion
        'worker_num'       => swoole_cpu_num() * 2,
```

will be more accurate

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
https://github.com/the-benchmarker/web-frameworks/actions/runs/3233130987/jobs/5294595318#step:6:994
I though we were using swoole 5.x

**Pull request merged**